interface/themes/core.css

```diff
@@ -76,38 +76,35 @@ button[href*="deleter"]:before {
     content: '\f053\00a0\00a0';
 }
 
+.btn-update::before {
+    content: '\f021\00a0\00a0';
```

Comment:
I am pretty sure we have already address this, but there is already a btn-update icon set in this script. Which would you prefer:
\f021
or
\f01e (this is the current one being used since it is called last on this script)